### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by cheempz**
* mention suppressing agent messages via APPOPTICS_LOG_SETTINGS from v10.0.0 release
* mention custom metrics are aggregated for AO-18250

### CODE REVIEW

**Review by prettycoder — APPROVED** [review]
lgtm :)

**Comment by cheempz** [comment]
LOL, I realized I was happily running `npm docs` per the contribution notes but it wasn't actually generating the documentation that should have been updated based on the code comment changes!

Looked into this a bit and we should be using `npm run "script action"` so updated the instructions and renamed the script action to be a bit more clear.

Please re-review @prettycoder :)

File: lib/api.js [hunk 1]
```diff
@@ -1334,6 +1334,8 @@ function sendMetric (name, options) {
  *                  value is associated with this type)
  * 2) value-based - a specific value (or sum of values if count > 1).
  *
+ * The metrics submitted are aggregated by metric name and tag(s), then
+ * sent on interval every 60 seconds.
```
Comment (prettycoder) [inline lib/api.js:1338]: `on interval` needed? I think the sentence would be fine just saying "The metrics submitted are aggregated by metric name and tag(s), then sent every 60 seconds."

**Review by prettycoder — APPROVED** [review]
lgtm :)
minor suggestion

File: lib/api.js (hunk 1 above)
Comment (cheempz) [reply lib/api.js]: That's true!  Updated.

**Pull request merged**